assistant/src/test/java/com/ibm/watson/assistant/v1/AssistantServiceIT.java

```diff
@@ -1770,7 +1770,7 @@ public void testDeleteUserData() {
   /** Test list mentions. */
   @Test
   public void testListMentions() {
-    String entity = "holiday";
```

Comment:
Why the change from holiday to beverage?

Reply:
it was failing, I'm not sure if something got deleted in the assistant